### ISSUE TITLE
OpenCTI docker installation does not work 

### ISSUE THREAD

**Issue by rocheston** [OpenCTI-Platform/docker#81]
After following the docker installation instructions from the website and correctly configuring the .env variables, I was able to log in to the dashboard; however, I was unable to get any of the connectors to connect to OpenCTI. I tried both the python script and docker-compose connector instances, making sure all the UUID variables were set correctly. Despite trying numerous combinations, nothing worked.

![grab6](https://user-images.githubusercontent.com/16910669/229283457-f05e13c4-ca15-4ec2-bf98-3738f23d9726.jpg)

The connectors are failing: OpenCTI API is not reachable. Why is your docker-compose file pointing the connector to http://opencti:8080? I changed to http://localhost:8080 does not work. I changed to http://172.17.0.1:8080 does not work or the host ip does not work. I tried every connector Mitre, AlienVault, all of the have the same issue.

![grab5](https://user-images.githubusercontent.com/16910669/229282783-4b9303bf-f27e-4f64-9104-c6c74d442c75.jpg)

Configuring OpenCTI on dockers has been a nightmare. Your instructions do not work, and it is unfortunate that I am unable to use the beautiful GUI that OpenCTI has to offer.

Please update your docker instructions and relevant yml configurations so that it simply works out of the box.

**Comment by cxfajar** [comment]
Could you share code of your Docker compose yml file so I can identify the issue.

**Comment by rocheston** [comment]
When using the docker-compose file from the OpenCTI-Platform/docker repository, any connectors added will generate an error stating 'OpenCTI API is not reachable'. This issue can be replicated by running the below docker-compose file. Please try it.

https://github.com/OpenCTI-Platform/docker/blob/master/docker-compose.yml

**Comment by cxfajar** [comment]
All the connectors that are given in this file are connectors for internal
export files; you have to add connectors from external imports like MISP
ALIENVAULT so the total number of entities and relationships will show up.
Here is a link where you can manually add any connector from an external
import:
https://github.com/OpenCTI-Platform/connectors/tree/master/external-import,
and I have a file that has already added some connectors. Here is the link
for that file:
https://github.com/2004gixxer600/openctihowto/blob/main/docker-compose.yml

Furthermore, if you are confused about API, you have to sign up on a
website like AlienVault, and from there you can copy your own API key and
paste it.

**Comment by rocheston** [comment]
Yes. I did all that that. No go. Even the internal connectors do not work.

**Comment by cxfajar** [comment]
I think connectors are working. Did you check on the dashboard? On the left
side there is a database symbol, and on top there is an option for
connectors. By clicking on it, you can see whether your connectors are
working or not. Actually, I am using Portainer for managing Docker
composer, so I have not had any issues like that.

**Comment by rocheston** [comment]
Does not work. I gave up on this tool. Useless.
